packages/protocol/contracts/L1/libs/LibProving.sol

```diff
@@ -19,6 +19,11 @@ library LibProving {
     /// @notice Keccak hash of the string "RETURN_LIVENESS_BOND".
     bytes32 public constant RETURN_LIVENESS_BOND = keccak256("RETURN_LIVENESS_BOND");
 
+    /// @notice Keccak hash of the string "tier_provider".
+    bytes32 private constant TIER_PROVIDER = bytes32("tier_provider");
```

Comment:
@dantaik how we have a LibConstants.sol to make include all these constants there?

Reply:
lets only put string constants there, other constants should stay where they are.

In LibConstants (maybe LibStrings?):, we can have naming convension, 

for a string "str", the variable is named S_STR, with "str" converted to upper case with a leading "S_"?

Reply:
We dont really have string constants, except 2: `PROVER_ASSIGMENT` or `SIGNAL`. Rest is byte32 representation of string.

Reply:
Same, same.